src/restview/restviewhttp.py

```diff
@@ -26,6 +28,10 @@
 __version__ = '3.0.3.dev0'
 
 
+CONFIG_FILE_PATH = Path.home() / '.restview.ini'
```

Comment:
Can we use `~/.config/restview/restview.ini` instead?  I would probably spell it
```suggestion
CONFIG_FILE_PATH = Path('~/.config/restview/restview.ini').expanduser()
```

Reply:
Done. Also, creates ~/.config/restview if it doesn't already exist.